basis/Holmakefile

```diff
@@ -1,19 +1,17 @@
-INCLUDES = ../developers ../misc pure ../translator ../characteristic ../semantics ../translator/monadic
-OPTIONS = QUIT_ON_FAILURE
+INCLUDES = $(CAKEMLDIR)/developers $(CAKEMLDIR)/misc\
+					 $(CAKEMLDIR)/semantics $(CAKEMLDIR)/characteristic\
+					 $(CAKEMLDIR)/translator $(CAKEMLDIR)/translator/monadic\
+					 pure
 
-all: $(DEFAULT_TARGETS) README.md
+CLINE_OPTIONS = --qof
```

Comment:
Don't need / shouldn't include `--qof` anymore:

```suggestion
 
```

Reply:
Grr; looks like proposed change would leave you with a line containing just `C`. How annoying.